app/assets/javascripts/calendars/company.es6

```diff
@@ -6,12 +6,16 @@ class CompanyCalendar extends Calendar {
       defaultView: 'agendaDay',
       resourceLabelText: 'Guiders',
       header: {
-        right: 'filter agendaDay timelineDay today jumpToDate prev,next'
+        right: 'sort filter agendaDay timelineDay today jumpToDate prev,next'
```

Comment:
We could probably just make this a top-level nav thing. But I'm not sure where it would go.

Hmm.